packages/fscomponents/src/components/Accordion.tsx

```diff
@@ -221,6 +225,13 @@ export class Accordion extends Component<AccordionProps, AccordionState> {
     };
   }
 
+  componentDidUpdate(prevProps: AccordionProps, prevState: AccordionState): void {
+    if (!!this.props.onStateChange && prevState.isOpen !== this.state.isOpen) {
```

Comment:
Function name has to help us determine purpose of the function. It is misleading to call a function `onStateChange` and then just compare `isOpen` flag.
Better name would be something like `onAccordionToggle` and pass `isOpen` flag to a callback.